schemas/dependencies/glitchtip-project-1.yml

```diff
@@ -91,8 +91,9 @@ properties:
       # referenced
       - "$ref": "/common-1.json#/definitions/crossref"
         "$schemaRef": "/dependencies/glitchtip-project-alert-1.yml"
-  acceptEvents:
-    type: boolean
+  eventThrottleRate:
+    type: integer
+    description: Percentage of new events to deny. 0-100
```

Comment:
Do we need schema limit `minimum` and `maximum`?

Reply:
Good idea. I added the limits in [8758fd1](https://github.com/app-sre/qontract-schemas/pull/586/commits/8758fd1bc084ba56da541994432ff5309a8917ff)